.github/dependabot.yml

```diff
@@ -11,3 +11,31 @@ updates:
     open-pull-requests-limit: 5
     schedule:
       interval: "daily"
+
+  # PROBLEM: 
+  # ========
+  # Since Dependabot does not have a way for ignoring sub directories (workspaces) yet in our monorepo.
+  # We'd allow those forked repos to be only updated once every month.
+  # The intention is just to limit the number of PRs coming from those subdirectories as possible. 
+  # - Link issue from Dependabot: (https://github.com/dependabot/dependabot-core/issues/4364)
+  # - Link bug reported issue from CryptKeeper: (https://github.com/CryptKeeperZK/crypt-keeper-extension/issues/667)
+  # 
+  # NOTE: 
+  # =====
+  # ALL the PRs from these subdirectories packages should be CLOSED directly.
+  # /packages/
+  #   proofs/
+  #     rln-proof/*
+  #     semaphore-proof/*
+  #   tools/
+  #     snarkjs/*
+  #     ffjavascript/*
```

Comment:
It would be better to move these forks away from current repo. It won't work if packages have the same deps with increase version strategy (for instance, prettier or eslint). It will update all the packages and anyway you need to fix it manually. In addition, you need to close all PRs related to these forks which requires you to take a look at each PR and do it manually.

The experiment with forks inside the monorepo shows that it doesn't work:
1. You need to update the forks manually here with copy-paste while in separate repo it can be done with github automation
2. It breaks the dependabot updates and the automation process

The only one good thing is it's located in monorepo but it creates more problems than it solves.

Reply:
Thanks @0xmad for your review, I just wrote a detailed explanation since I thought about this idea of separate repos. Let me know your thoughts about it.

Reply:
>It would be hard to fork only the @semphore-protocol/proof package only in a separate repo, because the original repo of the Semaphore project is also a monorepo and the proof package is part of this monorepo. That's why initially thought of moving only this proof package in our monorepo.

What's the problem to fork the whole semaphore?

>Not guaranteed that would resolve the initial problem of Dependabot which is:

I think it's related to ssh key for github action and it can be resolved via adding this key for workflows.
I think it possible can help to resolve it https://www.webfactory.de/blog/use-ssh-key-for-private-repositories-in-github-actions

Reply:
> What's the problem to fork the whole semaphore?
The Semaphore main package is a monorepo (https://github.com/semaphore-protocol/semaphore) and in our case we would only need to fork just one repo from this monorepo `packages/proofs`. So if we forked the whole Semaphore repo, it would be easier to ofc update it with the original but harder to only just export only this specific package and add it in our `package.json`. I guess we would need to publish it so we could only pick `@semaphore-protocol/proof`.

Reply:
Yes, we can publish fork packages in our scope. And It will fix the first issue you mentioned.